### DISCUSSION TITLE
Announcing .NET MAUI Preview 11?

### DISCUSSION THREAD

**Discussion by nogginbox** [dotnet/maui#3781] [General] [answered]
I can't find an announcement, but preview 11 seems to be out.

I was able to upgrade by running the following dotnet command:

```dotnet workload update```

That upgraded various things and I am now on 6.0.101-preview.11.2349

Actually, that only seemed to work till I rebooted Visual Studio. I also needed to run:

```dotnet workload install maui```

My app recompiled fine and it's fixed a few issues I was having too. So, pretty happy so far.

Would be nice to know what's new? Are there release notes?

**Comment by jfversluis** [comment, marked as answer]
Hey @NogginBox! Normally a new preview would come together with a new version of Visual Studio, however there seems to be something going on there. You are definitely able to get the new preview through the way you've described. Not sure what will happen with the blog post formally announcing it, but I managed to get my hands on a list of changes that are included. Have a look here: https://gist.github.com/Redth/347292446cc1d7f3bd381a2acae70c11

Or just compare the git tags with ALL the commits like this: https://github.com/dotnet/maui/compare/6.0.101-preview.10...6.0.101-preview.11.3

**Reply by drasticactions** [reply]
Preview 11 was shipping to Nuget with the anticipation that of a new Visual Studio 2022 17.1 preview also being released this week. Unfortunately, due to issues unrelated to MAUI, that release got delayed, leaving the MAUI release in limbo. 

I know that if you do upgrade to Preview 11 via the workloads, you will have issues with XAML Hot Reload. There were breaking changes for Dispatcher in MAUI that we had to match for XAML Hot Reload, and those changes were intended for VS2022 17.1 Preview 2... which is now delayed. That makes the current XAML Hot Reload released in VS2022 D17.1 Preview 1.1 incompatible with MAUI Preview 11.

As long as you know that, the rest of the development tooling _should_ work. But you'll most likely need to turn XAML Hot Reload off until D17.1 Preview 2 ships.

**Comment by emceelovin** [comment]
can confirm everything works fine(so far). not having hot reload isn't a dealbreaker for me at all, but i'm a little upset some stuff people, myself included, have submitted(some of which was tirag'd), hasnt been fixed :'(

**Reply by jfversluis** [reply]
It's being worked on! We can't fix everything at once :)

**Reply by emceelovin** [reply]
it's Christmas time. you guys need to hire more elves!
